Symfony/CS/Tests/Fixer/Symfony/NoEmptyStatementFixerTest.php

```diff
@@ -0,0 +1,434 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author SpacePossum
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+class NoEmptyStatementFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideNoEmptyStatementsCases
+     */
+    public function testNoEmptyStatements($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideNoEmptyStatementsCases()
+    {
+        return array(
+            array(
+                '<?php
+                use function Functional\map;
+                subnamespace\foo();
+                $a = new class {
+                    public function log($msg)
+                    {
+                    }
+                };
+                while($time < $a)
+                    ;
+                echo "done waiting.";
+                $b = \Test;
+                ',
+            ),
+            array(
+                '<?php
+                {
+                    '.'
+                }
+                echo 1;
+                ',
+                '<?php
+                {
+                    ;
+                };
+                echo 1;
+                ',
+            ),
+            array(
+                '<?php
+                namespace Two {}
+                $a = 1; {
+                }
```

Comment:
indeed, thanks, why doesn't fail locally, we do lint the input and output right?